collector.config.yaml

```diff
@@ -11,11 +11,24 @@ receivers:
 processors:
   batch:
 
+  tail_sampling:
```

Comment:
Shouldn't we use the filter approach instead?

Reply:
I think the trace state policy is fine but maybe @mathnogueira has a different opinion

Reply:
the filter preprocessor is faster and doesn't require as much memory as this one. I mean, it's ok to use this here, but the filter would be better.

Reply:
Just changed it to use filter instead, seems to be working correctly!